sdk/identity/azure-identity/samples/azure_cli_credential.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <iostream>
+
+#include <azure/identity/azure_cli_credential.hpp>
+
+#include <azure/service/client.hpp>
+
+int main()
+{
+  try
+  {
+    // Step 1: Initialize Azure CLI Credential.
+    auto chainedTokenCredential = std::make_shared<Azure::Identity::AzureCliCredential>();
```

Comment:
why do you use the chained credential here instead of the TokenCredential?  (I think the TokenCredential is the base of the chaninedCredential, right?)

Reply:
it is the variable name, but good catch! Copy-paste error.